dataclients/kubernetes/kube.go

```diff
@@ -391,16 +392,20 @@ func (c *Client) loadAndConvert() ([]*eskip.Route, error) {
 		return nil, err
 	}
 	c.state = state
+	previousUpdate := c.lastUpdate
+	c.lastUpdate = time.Now()
 	c.mu.Unlock()
 
+	enableLogging := log.GetLevel() >= log.DebugLevel || time.Since(previousUpdate) > 1*time.Minute
```

Comment:
@AlexanderYastrebov where is the test for this :see_no_evil:

Reply:
I don't see it :)

Reply:
Added a proper test for it.

Reply:
See https://github.com/zalando/skipper/pull/2427 :facepalm: